### PR TITLE
Jan 21

### PR DESCRIPTION
**Pull request by DanielTurner**
* C4A-201 - Complaints and recalls - New content
* C4A-407 - Maintenance - New content
* C4A-415 - PDF Format
* C4A-417 - Distorted yes/no buttons in print
* C4A-411 - Update validation code to ES5 friendly code
* Additional code for deployment from development repo with CI for deployment and uat branches. *In progress*

### CODE REVIEW

File: src/resources/complaints-and-recalls/questions/question-3/question.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<small class="hint">
+  For example:
+  <ul>
+    <li>Needles in strawberries</li>
+    <li>Rat poison at a buffet</li>
+    <li>Extorsion claim is made</li>
```
Comment (ThrawnCA) [inline src/resources/complaints-and-recalls/questions/question-3/question.html:6]: Misspelled "extortion"

File: src/resources/complaints-and-recalls/PDF/questions/question-3/question.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<small class="hint">
+  For example:
+  <ul>
+    <li>Needles in strawberries</li>
+    <li>Rat poison at a buffet</li>
+    <li>Extorsion claim is made</li>
```
Comment (ThrawnCA) [inline src/resources/complaints-and-recalls/PDF/questions/question-3/question.html:6]: Misspelled "Extortion"

File: src/resources/complaints-and-recalls/summary/section-one/view.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
+<link rel="stylesheet" href="./styles.scss" />
+
+<h1>Summary</h1>
+
+<p>
+  Thank you for taking the time to complete the Health and hygiene
```
Comment (ThrawnCA) [inline src/resources/complaints-and-recalls/summary/section-one/view.html:6]: Inconsistent capitalisation; "Health" is capitalised but "hygiene" is not.

File: src/resources/maintenance/PDF/questions-maintenance/question-3/yes.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<div class="alert alert-info" role="alert">
+  <h2><i class="fa fa-exclamation-triangle"></i>Warning!</h2>
+  <p>
+    You must not use any utensils or equipment that is chipped, cracked or
```
Comment (ThrawnCA) [inline src/resources/maintenance/PDF/questions-maintenance/question-3/yes.html:4]: "utensils or equipment that is" should be "utensils or equipment that are"

File: src/resources/maintenance/PDF/questions-maintenance/question-3/yes.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    broken. Germs can get into the damaged area. This may contaminate food and
+    make your customers sick.
+  </p>
+  <p>You should throw out any utensils and equipment that is damaged.</p>
```
Comment (ThrawnCA) [inline src/resources/maintenance/PDF/questions-maintenance/question-3/yes.html:8]: "utensils and equipment that is" should be "utensils or equipment that are"

File: src/resources/maintenance/questions/question-3/yes.html [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<div class="alert alert-info" role="alert">
+  <h2><i class="fa fa-exclamation-triangle"></i>Warning!</h2>
+  <p>
+    You must not use any utensils or equipment that is chipped, cracked or
```
Comment (ThrawnCA) [inline src/resources/maintenance/questions/question-3/yes.html:4]: "utensils or equipment that is" should be "utensils or equipment that are"

File: src/resources/maintenance/questions/question-3/yes.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+    broken. Germs can get into the damaged area. This may contaminate food and
+    make your customers sick.
+  </p>
+  <p>You should throw out any utensils and equipment that is damaged.</p>
```
Comment (ThrawnCA) [inline src/resources/maintenance/questions/question-3/yes.html:8]: "utensils and equipment that is" should be "utensils or equipment that are"

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

**Comment by duttonw** [comment]
I've created a ticket for these items you have raised @ThrawnCA  
C4A-489

**Review by ThrawnCA — CHANGES_REQUESTED** [review]
Typos haven't been addressed.

**Review by duttonw — APPROVED** [review]

**Pull request merged**